pkg/runtime/transaction/trx_log.go

```diff
@@ -68,16 +69,19 @@ CREATE TABLE IF NOT EXISTS __arana_trx_log
 // TxLogManager Transaction log management
 type TxLogManager struct {
 	sysDB proto.DB
+	timer *time.Timer
 }
 
 // init executes create __arana_tx_log table action
-func (gm *TxLogManager) init() error {
+func (gm *TxLogManager) init(delay time.Duration) error {
```

Comment:
when should we call this method for init? I did not find any caller of the method.

Reply:
Inner function "init" has been changed to Outer function "Init". It should be Init In function of CreateTrxManager at once.